lib/cc/engine/markdownlint.rb

```diff
@@ -33,7 +35,7 @@ def include_paths
         return root unless engine_config.has_key?("include_paths")
 
         markdown_files = engine_config["include_paths"].select do |path|
-          path.end_with?(".md")
+          EXTENSIONS.include?(File.extname(path)) || path.end_with?("/")
```

Comment:
Do you think this means `mdl` will try to analyze every file in these subdirectories? If so, do we need to pass `#{path}**/*.md` and `#{path}**/*.markdown` instead?